configs/container.config

```diff
@@ -0,0 +1,21 @@
+process {
+    withLabel: chromomap        { container = 'nanozoo/r_fungi:0.1--097b1bb' }
+    withLabel: deepvirfinder    { container = 'multifractal/deepvirfinder:0.1' }
+    withLabel: emboss           { container = 'quay.io/biocontainers/emboss:6.5.7--4' }
+    withLabel: ggplot2          { container = 'michelsteuwer/ggplot2:latest' }
+    withLabel: hmmscan          { container = 'mhoelzer/hmmscan:0.1' }
+    withLabel: marvel           { container = 'multifractal/marvel:0.1' }
+    withLabel: metaphinder      { container = 'multifractal/metaphinder:0.1' }
+    withLabel: pprmeta          { container = 'multifractal/ppr-meta:0.1' }
+    withLabel: prodigal         { container = 'nanozoo/prodigal:2.6.3--2769024' }
+    withLabel: python           { container = 'nanozoo/template:3.8--ccd0653' }
+    withLabel: r_plot           { container = 'replikation/r-phage-plot:latest' }
+    withLabel: samtools         { container = 'nanozoo/samtools:latest' }
```

Comment:
better to add a version tag here, latest is ``1.9--76b9270``

Reply:
adding it